src/inspector/node_string.cc

```diff
@@ -85,6 +85,28 @@ double toDouble(const char* buffer, size_t length, bool* ok) {
   return d;
 }
 
+std::unique_ptr<Value> parseMessage(const std::string& message, bool binary) {
+  if (binary) {
+    return Value::parseBinary(
+        reinterpret_cast<const uint8_t*>(message.data()),
+        message.length());
+  }
+  return parseJSON(message);
+}
+
+ProtocolMessage jsonToMessage(String message) {
+  return message;
+}
+
+ProtocolMessage binaryToMessage(std::vector<uint8_t> message) {
```

Comment:
This should probably be

```suggestion
ProtocolMessage binaryToMessage(const std::vector<uint8_t>& message) {
```

?

Reply:
Not really, we are `std::move`-ing binary messages around. It is likely that we switch to `using std::unique_ptr<std::vector>` for clarity at some point.